apps/pie-microsite/src/_11ty/shortcodes/tokensTable/index.js

```diff
@@ -83,22 +85,13 @@ const buildGlobalTokenUsedElement = globalToken => {
 
 /**
  * Builds the overall token description element for each type of token. The description content differs based on the type of token.
- * @param {string} token the token value i.e. #000, #ffffff, #000|0.85 or #000000|0.85
- * @param {*} tokenType the type of token i.e. color, spacing, radius
  * @param {*} tokenMetadata the metadata for the token. data such as descriptions
  * @returns {string} - the description HTML string
  */
-const buildTokenDescriptionElement = (tokenType, tokenMetadata) => {
-    const tokenTypeBuilder = {
-        [tokenTypes.COLOR]: buildColorDescription,
-        default: () => (tokenMetadata.description
-            ? `<span class="c-tokensTable-tokenDescription">${tokenMetadata.description}</span>`
-            : '')
-    };
-
-    let description = tokenType in tokenTypeBuilder
-        ? tokenTypeBuilder[tokenType](tokenMetadata)
-        : tokenTypeBuilder.default(tokenMetadata);
+const buildTokenDescriptionElement = tokenMetadata => {
+    let description = tokenMetadata.description
```

Comment:
Please can we wrap this in a `deindentHTML` call for readability? Then you can add proper indentation to the html string.

Reply:
actually, might be worth testing - can you just add the indentation without that function call and see if it works? the entire output is wrapped in one so it might be fine